samples/helloworld/enc/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# Use the edger8r to generate C bindings from the EDL file.
+add_custom_command(OUTPUT helloworld_t.h helloworld_t.c
+  COMMAND openenclave::oeedger8r --trusted ${CMAKE_SOURCE_DIR}/helloworld.edl)
+
+add_executable(enclave enc.c ${CMAKE_CURRENT_BINARY_DIR}/helloworld_t.c)
+
+target_compile_definitions(enclave PUBLIC OE_API_VERSION=2)
+
+# Need for the generated file helloworld_t.h
+target_include_directories(enclave PRIVATE ${CMAKE_CURRENT_BINARY_DIR})
+
+target_link_libraries(enclave openenclave::oeenclave openenclave::oelibc)
```

Comment:
Ah I just updated the documentation for this in #1585 😄 Sorry about that, dependencies got changed underneath me!